widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidget.kt

```diff
@@ -23,7 +23,7 @@ interface EntryWidget {
      *
      * @return The entry widget view.
      */
-    fun createEntryWidgetView(context: Context): View
+    fun getView(context: Context): View
```

Comment:
I think with the new method name we should clarify that every time this method is called it creates a new instance of the view. This is a public method, right?

Reply:
@gugalo Could you please have a look if it's clear now cbaca298b6004c4efa9fc80e4988360a7cf65cc5 ?